packages/js/src/dashboard/app.js

```diff
@@ -75,67 +76,93 @@ const App = () => {
+				<SidebarNavigation.Mobile
+					openButtonId="button-open-dashboard-navigation-mobile"
+					closeButtonId="button-close-dashboard-navigation-mobile"
+					/* translators: Hidden accessibility text. */
+					openButtonScreenReaderText={ __( "Open dashboard navigation", "wordpress-seo" ) }
+					/* translators: Hidden accessibility text. */
+					closeButtonScreenReaderText={ __( "Close dashboard navigation", "wordpress-seo" ) }
+					aria-label={ __( "Dashboard navigation", "wordpress-seo" ) }
+				>
+					<Menu idSuffix="-mobile" />
+				</SidebarNavigation.Mobile>
+				<div className="yst-p-4 min-[783px]:yst-p-8 yst-flex yst-gap-4">
+					<aside className="yst-sidebar yst-sidebar-nav yst-shrink-0 yst-hidden min-[783px]:yst-block yst-pb-6 yst-bottom-0 yst-w-56">
+						<SidebarNavigation.Sidebar>
+							<Menu />
+						</SidebarNavigation.Sidebar>
+					</aside>
+					<div className="yst-grow">
+						<div>
+							{ shouldShowWebinarPromotionNotificationInDashboard( STORE_NAME ) &&
+								<WebinarPromoNotification store={ STORE_NAME } url={ webinarIntroSettingsUrl } image={ null } />
+							}
+							{ notices.length > 0 && <div className="yst-space-y-3 yoast-new-dashboard-notices"> {
+								notices.map( ( notice, index ) => (
+									<Notice
+										key={ index }
+										id={ notice.id || "yoast-dashboard-notice-" + index }
+										title={ notice.header }
+										isDismissable={ notice.isDismissable }
+									>
+										{ notice.content }
+									</Notice>
+								) )
+							}
+							</div> }
+						</div>
+						<div className="yst-space-y-6 yst-mb-8 xl:yst-mb-0">
+							<main>
+								<Transition
+									key={ pathname }
+									appear={ true }
+									show={ true }
+									enter="yst-transition-opacity yst-delay-100 yst-duration-300"
+									enterFrom="yst-opacity-0"
+									enterTo="yst-opacity-100"
 								>
-									{ notice.content }
-								</Notice>
-							) )
-						}
-						</div> }
-					</div>
-					<div className="yst-space-y-6 yst-mb-8 xl:yst-mb-0">
-						<main>
-							<Transition
-								key={ pathname }
-								appear={ true }
-								show={ true }
-								enter="yst-transition-opacity yst-delay-100 yst-duration-300"
-								enterFrom="yst-opacity-0"
-								enterTo="yst-opacity-100"
-							>
-								<Routes>
-									<Route path="/" element={ <AlertCenter /> } />
-									<Route path="/first-time-configuration" element={ <FirstTimeConfiguration /> } />
-								</Routes>
-							</Transition>
-						</main>
+									<Routes>
+										<Route path="/" element={ <AlertCenter /> } />
+										<Route path="/first-time-configuration" element={ <FirstTimeConfiguration /> } />
+									</Routes>
+								</Transition>
+							</main>
+						</div>
 					</div>
 				</div>
-			</div>
-		</SidebarNavigation>
+			</SidebarNavigation>
+			<Notifications
+				className="yst-mx-[calc(50%-50vw)] yst-transition-all lg:yst-left-44"
```

Comment:
I'm not entirely sure what these classes are supposed to do 🤔 
We do have styling for the notification center in the stylesheet that is supposed to take all the WP menu stuff into consideration.

Reply:
For some reason the style sheet does not work... I investigated it a bit but couldn't find any clue. However I removed the useless `yst-transition-all` class.